### PR TITLE
Add mail service support

### PR DESCRIPTION
**Pull request by MarchenkoVladislav**

### CODE REVIEW

**Comment by SHvatov** [comment]
Modified tests a little. I suggest we use JUnit 5, instead of 4 and its extensions and so on, because they are more useful and can be easily reused or configured the way we want it to be.

File: src/test/kotlin/com/discounts/nearby/config/MailServiceTestConfig.kt [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package com.discounts.nearby.config
```
Comment (SHvatov) [inline src/test/kotlin/com/discounts/nearby/config/MailServiceTestConfig.kt:1]: Modified by SHvatov, see comments

File: src/test/kotlin/com/discounts/nearby/config/extension/SmtpMailServerExtension.kt [hunk 2]
```diff
@@ -0,0 +1,50 @@
+package com.discounts.nearby.config.extension
```
Comment (SHvatov) [inline src/test/kotlin/com/discounts/nearby/config/extension/SmtpMailServerExtension.kt:1]: Modified by SHvatov, see comments

File: src/test/kotlin/com/discounts/nearby/service/MailServiceTest.kt [hunk 3]
```diff
@@ -0,0 +1,52 @@
+package com.discounts.nearby.service
```
[the remark this answers is not in the document]
Comment (SHvatov) [reply src/test/kotlin/com/discounts/nearby/service/MailServiceTest.kt:1]: Modified by SHvatov, see comments

File: src/test/resources/application-test.properties [hunk 4]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (SHvatov) [inline src/test/resources/application-test.properties:1]: Deleted properties cause they are not useful at the moment

**Review by SHvatov — CHANGES_REQUESTED** [review]

**Comment by SHvatov** [comment]
[Add support for the mail services](https://trello.com/c/lcNQ0B3Q/10-add-support-for-the-mail-services)

**Comment by MarchenkoVladislav** [comment]
I fixed code by your recommendations.

**Review by SHvatov — APPROVED** [review]

**Pull request merged**